python/paddle/nn/layer/common.py

```diff
@@ -58,7 +58,7 @@ def _npairs(x: _T_Padding, n: int) -> _T_Padding: ...
 
 
 @overload
-def _npairs(x: int, n: int) -> int: ...
+def _npairs(x: int, n: int) -> Sequence[int]: ...
```

Comment:
```suggestion
def _npairs(x: int, n: int) -> list[int]: ...
```

参数应尽可能使用抽象类型，返回值应尽可能使用具体类型[^1][^2]

[^1]: https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/style_guide_and_references/type_annotations_specification_cn.html#canshuyingjinkenengshiyongchouxiangleixing-fanhuizhiyingjinkenengshiyongjutileixing
[^2]: https://typing.python.org/en/latest/reference/best_practices.html#arguments-and-return-types

Reply:
学到了，感谢～